DisplayLarge.java

```diff
@@ -11,24 +11,25 @@
  * If the numbers are equal, print the message "These numbers are equal." 
  */
 public class DisplayLarge {
-	public static void main(String[] args) {
-		String number1 = JOptionPane.showInputDialog("Enter Number1");
-		String number2 = JOptionPane.showInputDialog("Enter Number2");
-		int num1 = Integer.parseInt(number1);
-		int num2 = Integer.parseInt(number2);
-		int large;
-		if (num1 > num2) {
-			large = num1;
-		}
-		else {
-			large = num2;
-		}
-		if (num1 == num2) {
-			JOptionPane.showMessageDialog(null, "These numbers are equal");
-		}
-		else {
-			JOptionPane.showMessageDialog(null, large + " is larger");
-		}
+	public int[] input() {
+		int inputs[] = new int[2];
+		for (int i = 0; i < 2; i++) Integer.parseInt(JOptionPane.showInputDialog("Enter Number" + ++i));
```

Comment:
To which value Integer.parseInt(JOptionPane.showInputDialog("Enter Number" + ++i)) being assinged?